x-pack/plugins/apm/public/components/app/ServiceOverview/NoServicesMessage.tsx

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiEmptyPrompt } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { KibanaLink } from '../../shared/Links/KibanaLink';
+import { SetupInstructionsLink } from '../../shared/Links/SetupInstructionsLink';
+
+interface Props {
+  hasDataEver: boolean;
+}
+
+export function NoServicesMessage({ hasDataEver }: Props) {
+  if (hasDataEver) {
+    return (
+      <EuiEmptyPrompt
+        title={
+          <div>
+            {i18n.translate('xpack.apm.servicesTable.notFoundLabel', {
+              defaultMessage: 'No services found'
+            })}
+          </div>
+        }
+        titleSize="s"
+      />
+    );
+  } else {
+    return (
+      <EuiEmptyPrompt
+        title={
+          <div>
+            {i18n.translate('xpack.apm.servicesTable.noServicesLabel', {
+              defaultMessage: `Looks like you don't have any APM services installed. Let's add some!`
+            })}
+          </div>
+        }
+        titleSize="s"
+        body={
+          <React.Fragment>
+            <p>
+              {i18n.translate(
+                'xpack.apm.servicesTable.7xUpgradeServerMessage',
+                {
+                  defaultMessage: `Upgrading from a pre-7.x version? Make sure you've also upgraded
+              your APM server instance(s) to at least 7.0.`
+                }
+              )}
+            </p>
+            <p>
+              {i18n.translate('xpack.apm.servicesTable.7xOldDataMessage', {
+                defaultMessage:
+                  'You may also have old data that needs to be migrated.'
+              })}{' '}
+              {/* TODO: GET REAL URL FOR KIBANA MIGRATION ASSISTANT */}
+              <KibanaLink pathname="/app/kibana" hash="/management">
```

Comment:
I think this is the url: `/app/kibana#/management/elasticsearch/upgrade_assistant`.
Unfortunately we cannot link to the "Indices" tab: https://github.com/elastic/kibana/pull/29845

Reply:
Hm I saw that in the management section but it says 8.0 Upgrade Assistant even when I'm on the 7.0 branch ... I'm not sure who to confirm this with

Reply:
@tylersmalley would be the one to ask.

Reply:
Confirmed with the operations team that this is the correct URL. Will remove that comment and push.